Agreements/Airport/Mor_PB.txt

```diff
@@ -1 +1,3 @@
-COPX:*:*:SUM:EGPB:*:Moray:PBAPP:*:11000:|SUM
\ No newline at end of file
+COPX:*:*:*:EGPB:*:Moray:PBAPP:*:13000:|SUM
+COPX:*:*:*:EGPM:*:Moray:PBAPP:*:13000:|SUM
+COPX:*:*:*:EGET:*:Moray:PBAPP:*:13000:|SUM
```

Comment:
Same logic as below (I'd read the other comment before this one... I reviewed in a weird order), I think we should define a FIXBEFORE of ADN. I'm not 100% sure whether EuroScope logic works on the basis of ADN being in the route at _any_ point, or explicitly the last fix before entering the next sector in question... so it may be necessary to _also_ define agreements for FIXBEFORE BEKET/PETOX - this might need testing on sweatbox